mne/channels/channels.py

```diff
@@ -88,46 +88,86 @@ def _get_ch_type(inst, ch_type, allow_ref_meg=False):
 
 
 @verbose
-def equalize_channels(candidates, verbose=None):
-    """Equalize channel picks for a collection of MNE-Python objects.
+def equalize_channels(instances, copy=False, verbose=None):
```

Comment:
functions should copy by default. Only methods should not do this and operated inplace

Reply:
True. However, that would be change from current master, since that version operated in-place only. I'm being inconsistent here I see, since I break backwards compatibility anyway be re-ordering channels.

I could avoid breaking backwards compatibility here by making the signature:

```python
equalize_channels(candidates, copy=False, ordering=False, verbose=None)
```

if that is preferable. However, every time I would want to use the function, I want `copy=True, ordering=True` and don't see much scenarios where you wouldn't want that.

Reply:
No strong feeling but at least API change should be documented in what's new. The current function in master operates inplace and returns modified inputs which is evil...

Reply:
`copy=True` it is!